docs/Reference/Server.md

```diff
@@ -1479,7 +1479,8 @@ call is encapsulated by prefix, so different plugins can set different not found
 handlers if a different [`prefix` option](./Plugins.md#route-prefixing-option)
 is passed to `fastify.register()`. The handler is treated as a regular route
 handler so requests will go through the full [Fastify
-lifecycle](./Lifecycle.md#lifecycle). *async-await* is supported as well.
+lifecycle](./Lifecycle.md#lifecycle) for valid URLs. Invalid URLs are sent 
```

Comment:
I don't think this is accurate. If a path does not match any configured routes, the request is _not_ passed to a route handler. It is instead handled by the configured 404 handler.

Reply:
You are 💯 right, I meant to write "directly to the 404 handler" but that last part got removed. With that in place it would be accurate, right?

Reply:
Yes, I think that would be accurate.

Reply:
One things which I have failed to add here, is that in v5 there is a change in that "bad URLs" are no longer sent to the handler but must be dealt with by adding a [`.frameworkErrors`](https://fastify.dev/docs/latest/Reference/Server/#frameworkerrors) handler if you need to do handle them in a specific manner.